internal/api/handlers/schedulers_handler.go

```diff
@@ -82,6 +82,21 @@ func (h *SchedulersHandler) GetScheduler(ctx context.Context, request *api.GetSc
 	return &api.GetSchedulerResponse{Scheduler: h.fromEntitySchedulerToResponse(scheduler)}, nil
 }
 
+func (h *SchedulersHandler) GetSchedulerVersions(ctx context.Context, request *api.GetSchedulerVersionsRequest) (*api.GetSchedulerVersionsResponse, error) {
+	var versions []*entities.SchedulerVersion
```

Comment:
@caiordjesus Couldn't we declare and assign those vars using `versions, err :=` instead of making it separately?

Reply:
You're absolutely right!